service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -0,0 +1,153 @@
+package chartoperator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/helm/pkg/helm"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+const (
+	Name = "chartoperatorv1"
+
+	defaultClusterDNSIP    = "172.31.0.10"
```

Comment:
Adding a default value, but we surely have this value for newer cluster.

Reply:
Hmm yes, this is for bootstrapping CoreDNS and it should come from the cluster configmap. 

What happens if this is not set?

Reply:
Then it would use default value in here, https://github.com/giantswarm/chart-operator/blob/854a6b727ed86ee05e172d84bc6a03e8e1429ab7/helm/chart-operator-chart/values.yaml#L11

Therefore I can safely remove this value.